src/mpyl/project.py

```diff
@@ -395,6 +412,14 @@ def s3_bucket(self) -> S3Bucket:
             raise KeyError(f"Project '{self.name}' does not have s3 configuration")
         return self.deployment.s3_bucket
 
+    @property
+    def dagster(self) -> Dagster:
+        if self.deployment is None or self.deployment.dagster is None:
+            raise KeyError(
+                f"Project '{self.name}' does not have kubernetes configuration"
```

Comment:
dagster configuration instead of kubernetes configuration?

Reply:
done in
[34b4331](https://github.com/Vandebron/mpyl/pull/146/commits/34b4331b34b4fa1ada15dd88ce7115eb7e08464c)